index.html

```diff
@@ -1604,6 +1604,30 @@ <h3>Issuer</h3>
 example, <code>"did:example:abfe13f712120431c276e12ecab"</code>).
         </p>
 
+       <pre class="example nohighlight" title="Usage of issuer property with object value">
+{
+  "@context": [
+    "https://www.w3.org/2018/credentials/v1",
+    "https://www.w3.org/2018/credentials/examples/v1"
+  ],
+  "id": "http://example.edu/credentials/3732",
+  "type": ["VerifiableCredential", "UniversityDegreeCredential"],
+  <span class="highlight">"issuer": {
+      "id": "https://example.com/keys/foo.jwk",
+      "type": "jwk"
```

Comment:
We need to define this in the JSON-LD context or use something like urn:example:jwk.

Reply:
changed in [3c5b319](https://github.com/w3c/vc-data-model/pull/710/commits/3c5b319b41b618b831c127dfaefea3004ac72b16)

Reply:
Let me know if this change works.